ui/pages/Search/components/VariantSearchResultsNew.jsx

```diff
@@ -4,22 +4,34 @@ import { connect } from 'react-redux'
 import { Grid, Message, Button } from 'semantic-ui-react'
 import styled from 'styled-components'
 
-import { loadSearchedVariants, unloadSearchResults } from 'redux/rootReducer'
+import DataLoader from 'shared/components/DataLoader'
```

Comment:
this file is mostly the moved shared `VariantSearchResults` file with the logic from the search-page specific file integrated with it